docs/pages/management/export-audit-events/panther.mdx

```diff
@@ -0,0 +1,246 @@
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/7. Install the Event Handler plugin
+
+The Teleport event handler runs alongside the Fluentd forwarder, receives events
+from Teleport's events API, and forwards them to Fluentd.
+
+(!docs/pages/includes/install-event-handler.mdx!)
+
+## Step 2/7. Generate a plugin configuration
+
+(!docs/pages/includes/configure-event-handler.mdx!)
+
+## Step 3/7. Create a user and role for reading audit events
+
+(!docs/pages/includes/plugins/event-handler-role-user.mdx!)
+
+## Step 4/7. Create teleport-event-handler credentials
+
+In order for the Event Handler plugin to forward events from your Teleport cluster, 
+it needs signed credentials from the cluster's certificate authority. The teleport-event-handler 
+user cannot request this itself, and should use Teleport Machine ID to obtain the credentials.
+
+Teleport Machine ID bot should leverage the `teleport-event-handler` role to request the credentials.
+Machine ID already impersonates roles when requesting credentials from the Teleport cluster.  
+
+### Export an identity file for the Event Handler plugin user
+
+Give the plugin access to a Teleport identity file. We recommend using Machine
+ID for this in order to produce short-lived identity files that are less
+dangerous if exfiltrated, though in demo deployments, you can generate
+longer-lived identity files with `tctl`:
+
+<Tabs>
+<TabItem label="Machine ID">
+(!docs/pages/includes/plugins/tbot-identity.mdx secret="teleport-event-handler-identity"!)
+</TabItem>
+<TabItem label="Long-lived identity files">
+(!docs/pages/includes/plugins/identity-export.mdx user="teleport-event-handler" secret="teleport-event-handler-identity"!)
+</TabItem>
+</Tabs>
+
+## Step 5/7. Create a Dockerfile with Fluentd and the S3 plugin
+
+To send logs to Panther, you need to use the Fluentd output plugin for S3. Create
+a `Dockerfile` with the following content:
+
+```text
+FROM fluent/fluentd:edge
+USER root
+RUN fluent-gem install fluent-plugin-s3
+USER fluent
+```
+
+Build the Docker image:
+
+```shell
+$ docker build -t fluentd-s3 .
+```
+
+<Admonition type="tip" title="Testing Locally?">
+
+If you're running Fluentd in a local Docker container for testing, you can adjust
+the entrypoint to an interactive shell as the root user, so you can test the setup.
+
+```code
+$ docker run -u $(id -u root):$(id -g root) -p 8888:8888 -v $(pwd):/keys -v \
+$(pwd)/fluent.conf:/fluentd/etc/fluent.conf --entrypoint=/bin/sh -i --tty  fluentd-s3
+```
+
+</Admonition>
+
+### Configure Fluentd for Panther
+
+When you run the Teleport Event Handler, it will create a `fluent.conf` file.  This file needs to be updated
+to send logs to Panther. This means adding a `<filter>` and `<match>` section to the file. These sections
+will filter and format the logs before sending them to S3, The record_transformer is important to send the 
+right date and time format for Panther. 
+
+```xml
+<!--
```

Comment:
I would explain why we're commenting these out

Reply:
added an extra comment to explain that it's from the teleport-event-forwarder.